src/PearlLPCompounder.sol

```diff
@@ -154,6 +157,8 @@ contract PearlLPCompounder is BaseHealthCheck, CustomStrategyTriggerBase {
 
     /// @notice Set if we should use Curve AAVE pool for stable swaps
     /// @param _useCurveStable true if we should use Curve AAVE pool for stable swaps
+    // Review: when would this method be used?
+    // Doesn't the contructor take care of checking if a stable swap is possible?
```

Comment:
Constructor sets only `curveStableIndex` in curve 3 pool. With bool `useCurveStable` we can switch to use curve for swapping stables instead of synapse protocol.